papermill_origami/engine.py

```diff
@@ -124,11 +125,18 @@ def wrapper(cell, *args, **kwargs):
             for key, value in flatten_dict(
                 cell.metadata.papermill, parent_key_tuple=("papermill",)
             ).items():
-                run_sync(self.km.client.update_cell_metadata)(
-                    file=self.file,
-                    cell_id=cell.id,
-                    metadata_update_properties={"path": key, "value": value},
-                )
+                try:
+                    run_sync(self.km.client.update_cell_metadata)(
+                        file=self.file,
+                        cell_id=cell.id,
+                        metadata_update_properties={"path": key, "value": value},
+                    )
+                except (
+                    asyncio.exceptions.TimeoutError,
+                    websockets.exceptions.ConnectionClosedError,
+                ):
+                    logger.debug("Encountered an error while updating cell metadata")
+                    pass
```

Comment:
Why pass on the exception? I could see reraising as a new exception but just ignoring it seems potentially problematic

Reply:
`update_cell_metadata` is not a critical code path and the feature to sync metadata with the Noteable notebook was added as a nice-to-have. Noteable UI does **not** use the `papermill` metadata that papermill-origami sends over RTU.

Raising the exception here would cause the papermill execution loop to break and we wouldn't want that due to a failure here -- is my opinion.